Add 2.1.0.rc1 announcement post

timriley opened the pull request:
Ready for you to review, @jodosha :)

When it comes to publishing this, **don't forget to update the `date:` in the front matter!** 😜

---

Closes https://github.com/hanami/hanami.github.io/issues/579

timriley committed:
Add 2.1.0.rc1 announcement post

waiting-for-dev commented on source/blog/2023-10-24-hanami-210rc1.html.markdown:
```diff
@@ -0,0 +1,140 @@
+
+After beta2, we made some refinments to our assets helper names to minimise naming collisions in your views. The adjusted names are `javascript_tag`, `stylesheet_tag`, `favicon_tag`, `image_tag`, `video_tag` and `audio_tag`.
+
+In addition, helpers inside your view part classes are now contained inside a single `helpers` object, ensuring no collisions with the many method names that view parts forwrd to your app’s domain objects. Accessing a helper inside a view part new looks like this:
```
```suggestion
In addition, helpers inside your view part classes are now contained inside a single `helpers` object, ensuring no collisions with the many method names that view parts forward to your app’s domain objects. Accessing a helper inside a view part new looks like this:
```

timriley replied:
Thanks! Pushed a fix for this.

waiting-for-dev reviewed (approved):
Awesome content! I only found a typo :slightly_smiling_face:

timriley committed:
svelte->stylish

This may have led to some confusion given the fact that a JS framework called “Svelte” exists.

timriley committed:
Fix typos

timriley commented:
n.b. at this point we'll need to change both the file name _and_ the date before posting (since we delayed this by a week).

In the meantime, this let me look at the deploy preview today, and it seems like the images in the post are 404ing. Is there a different place I need to put them, @jodosha?

timriley committed:
Update date for post

timriley commented:
Have just pushed updates to this anticipating the new release date: 2023-11-01

jodosha committed:
Last details

jodosha committed:
Fixed image paths

jodosha committed:
Advertise the announcement in home page

jodosha reviewed (approved):
@timriley Ready to go! 🏁
